### ISSUE TITLE
Click on background to close gallery not working?

### ISSUE THREAD

**Issue by q2apro** [sachinchoolur/lightGallery#277]
I updated from 1.1.5 to 2 latest and see that the click on background to close the gallery does not work anymore. 

I cannot find an option to enable this again. Can someone point me to it?

I would also be interested what changed from the one version to the other one. The design seems different now. I also want to get rid of the new bar on top. Thanks.

**Comment by EnriqueCanals** [comment]
I'm also experiencing this issue, but only on native mobile devices. Desktop browsers do properly dismiss the gallery on background click but not mobile browsers.

**Comment by Yonben** [comment]
Experiencing the same bug here. Any fix ?

**Comment by xd0** [comment]
I'm having the same issue with lightGallery 1.2.19 that when i click on "backdrop" (on iPad) it doesn't close the gallery. Is there any fix for this?

**Comment by philipp-unger** [comment]
$('.lg-backdrop').bind('click touchend', function () {
   $lg.data('lightGallery').destroy(false);
 });

**Comment by sachinchoolur** [comment]
Added swipe up/down option to close the gallery. Will be released with 2.0

**Comment by sachinchoolur** [comment]
Fixed in version 1.6.7 -  https://github.com/sachinchoolur/lightGallery/releases/tag/1.6.7

**Comment by marcelogarbin** [comment]
@sachinchoolur could you help with this? when I open a gallery image in chrome when clicking on the overlay it is not closing. The version I use is: `lightgallery@1.6.12`

my code: 
```javascript
$(document).ready(function() {
    $(".lightgallery").lightGallery({
        selector: '.item',
        share: false,
        showThumbByDefault: false,
        thumbnail: false,
        zoom: false,
        fullScreen: false,
        closable: true,
        youtubePlayerParams: {
            modestbranding: 1,
            showinfo: 0,
            rel: 0,
            controls: 2
        },
        vimeoPlayerParams: {
            byline : 0,
            portrait : 0
        }
    });
});
```
